### PR TITLE
Fix NativeAOT JIT-EE bug

### PR DESCRIPTION
**Pull request by MichalPetryka**

### CODE REVIEW

File: src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs [hunk 1]
```diff
@@ -2784,9 +2784,7 @@ private CorInfoType getChildType(CORINFO_CLASS_STRUCT_* clsHnd, CORINFO_CLASS_ST
                 result = asCorInfoType(returnType, clsRet);
             }
             else
```
Comment (EgorBo) [inline src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs:2786]: can you wrap it with { } because it looks a bit off

**Review by EgorBo — APPROVED** [review]
Nice catch

File: src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs (hunk 1 above)
Comment (MichalPetryka) [reply src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs:2786]: Done.

**Review by jkotas — APPROVED** [review]
Thank you!

**Pull request merged**